src/util/DateLocale.php

```diff
@@ -65,9 +77,9 @@ public function Set($aLocale)
         }
 
         for ($i = 1; $i <= 12; ++$i) {
-            list($short, $full)            = explode('|', strftime('%b|%B', strtotime("2001-${i}-01")));
-            $this->iShortMonth[$aLocale][] = ucfirst($short);
-            $this->iMonthName[$aLocale][]  = ucfirst($full);
+            list($short, $full)                = explode('|', strftime('%b|%B', strtotime("2001-${i}-01")));
```

Comment:
strftime is deprecated as of php8.1 https://www.php.net/manual/en/function.strftime.php